website/src/components/TutorialComponents/CodeBlock.tsx

```diff
@@ -61,7 +107,7 @@ function CodeEditor(props: {
     }, []);
 
     useEditable(editorRef, onEditableChange, {
-        disabled: props.disabled,
+        disabled: disabled,
```

Comment:
nit:

    disabled: disabled => disabled

Reply:
What's the problem with using the boolean directly?

Reply:
it's just a TypeScript shorthand, where you can put the variable and TypeScript will use the variable name as the field name.

    { disabled: disabled }

to
    {disabled}

Reply:
nit = nit picking